crates/aptos-crypto/src/ed25519/ed25519_sigs.rs

```diff
@@ -18,6 +18,15 @@ use std::{cmp::Ordering, fmt};
 #[derive(DeserializeKey, Clone, SerializeKey)]
 pub struct Ed25519Signature(pub(crate) ed25519_dalek::Signature);
 
+#[cfg(any(test, feature = "fuzzing"))]
+impl<'a> arbitrary::Arbitrary<'a> for Ed25519Signature {
+    fn arbitrary(u: &mut arbitrary::Unstructured<'a>) -> arbitrary::Result<Self> {
```

Comment:
Same point here as for the PK.

It's even trickier here because a "valid" signature is defined w.r.t. to a PK and a message, so you cannot even sample it randomly without having those.